pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -196,6 +196,7 @@ enum ServerError {
 
     TransactionCoordinatorNotFound = 20; // Transaction coordinator not found error
     InvalidTxnStatus = 21; // Invalid txn status error
+    NotAllowedError = 22; // Not allowed error
```

Comment:
From the ci error. we need to handle this new error type in `ClientConnection.cc`